api/src/pcapi/scripts/beneficiary/import_users.py

```diff
@@ -3,19 +3,77 @@
 import logging
 import sys
 from typing import Iterable
+from typing import Optional
 
 from pcapi import settings
 import pcapi.core.payments.api as payments_api
 import pcapi.core.users.api as users_api
+from pcapi.core.users.models import EligibilityType
 from pcapi.core.users.models import User
+from pcapi.core.users.models import UserRole
 from pcapi.core.users.repository import find_user_by_email
 from pcapi.core.users.utils import sanitize_email
 from pcapi.repository import repository
+from pcapi.routes.serialization.users import ProUserCreationBodyModel
 
 
 logger = logging.getLogger(__name__)
 
 
+def _get_birth_date(row: dict) -> datetime:
+    return datetime.strptime(row["Date de naissance"], "%Y-%m-%d")
+
+
+def _create_user(row: dict, role: Optional[UserRole]) -> User:
```

Comment:
(détail) pour être plus explicite :

```suggestion
def _create_beneficiary(row: dict, role: Optional[UserRole]) -> User:
```